karma.conf.js

```diff
@@ -7,10 +7,10 @@ let customLaunchers = {
     base: 'ChromeHeadless',
     flags: ['--disable-gpu', '--no-sandbox'],
   },
-  FirefoxHeadless: {
-    base: 'Firefox',
-    flags: ['-headless'],
-  },
+  // FirefoxHeadless: {
+  //   base: 'Firefox',
+  //   flags: ['-headless'],
+  // },
```

Comment:
Don't particularly like this, since this affects testing outside jenkinsx. I'd rather have the tests running on Chrome via cli, wdyt? ⬇️